src/Umbraco.Web.BackOffice/Controllers/MemberController.cs

```diff
@@ -286,172 +308,256 @@ public MemberDisplay GetEmpty(string contentTypeAlias = null)
                     if (destProp != null)
                     {
-                        //if found, change the value of the contentItem model to the persisted value so it remains unchanged
-                        var origValue = contentItem.PersistedContent.GetValue(sensitiveProperty.Alias);
+                        // if found, change the value of the contentItem model to the persisted value so it remains unchanged
+                        object origValue = contentItem.PersistedContent.GetValue(sensitiveProperty.Alias);
                         destProp.Value = origValue;
                     }
                 }
             }
 
-            var isLockedOut = contentItem.IsLockedOut;
+            bool isLockedOut = contentItem.IsLockedOut;
 
-            //if they were locked but now they are trying to be unlocked
+            // if they were locked but now they are trying to be unlocked
             if (contentItem.PersistedContent.IsLockedOut && isLockedOut == false)
             {
                 contentItem.PersistedContent.IsLockedOut = false;
                 contentItem.PersistedContent.FailedPasswordAttempts = 0;
             }
             else if (!contentItem.PersistedContent.IsLockedOut && isLockedOut)
             {
-                //NOTE: This should not ever happen unless someone is mucking around with the request data.
-                //An admin cannot simply lock a user, they get locked out by password attempts, but an admin can un-approve them
+                // NOTE: This should not ever happen unless someone is mucking around with the request data.
+                // An admin cannot simply lock a user, they get locked out by password attempts, but an admin can un-approve them
                 ModelState.AddModelError("custom", "An admin cannot lock a user");
             }
 
-            //no password changes then exit ?
-            if (contentItem.Password == null)
-                return;
+            MembersIdentityUser identityMember = await _memberManager.FindByIdAsync(contentItem.Id.ToString());
+            if (identityMember == null)
+            {
+                throw HttpResponseException.CreateNotificationValidationErrorResponse("Member was not found");
+            }
 
-            throw new NotImplementedException("Members have not been migrated to netcore");
-            // TODO: all member password processing and creation needs to be done with a new aspnet identity MemberUserManager that hasn't been created yet.
-            // set the password
-            //contentItem.PersistedContent.RawPasswordValue = _passwordSecurity.HashPasswordForStorage(contentItem.Password.NewPassword);
-        }
+            if (contentItem.Password != null)
+            {
+                IdentityResult validatePassword = await _memberManager.ValidatePasswordAsync(contentItem.Password.NewPassword);
```

Comment:
@Shazwazza @bergmania after a fair amount of debugging I do need a bit of assistance with this. The same issue is happening here in `UpdateMemberAsync` and `ValidateMemberDataAsync` below.

This uses the new `ValidatePasswordAsync` method on Identity member manager for our the validation of a password to determine whether the password is valid. 

- If valid, we hash the password and add it to the identity object to be updated as part of the update result.
- If the password does not validate according the the `PasswordValidators`, I thought we could create a suitable validation error response with the reasons. Something like this (although the formatting of the error message may need work):

![image](https://user-images.githubusercontent.com/1649855/104350463-140fbe80-54fc-11eb-856d-f690a5283eab.png)

In this branch, if the password doesn't validate, the error message does displays in the back-office as expected. 

However, my issue is that after this, upon clicking the field it never disappears (unlike other validation messages). I think I'm missing something I need to do in terms of what I pass to the client-side code?

This doesn't happen with users since the password isn't validated in this way, as it is generated rather than manually entered. Other places where a password is manually entered seem to return a slightly different validation object.

I also saw some work happening around validation recently so I didn't want to go too far trying to debug this if it is not the approach we're looking for anyway. 

Any help appreciated!

Reply:
Seems like an issue introduced in this PR: https://github.com/umbraco/Umbraco-CMS/pull/7263/. I will see if I can find it

Reply:
@bergmania possibly, although I introduced this password validation functionality so it is hard to tell whether it was already happening. Does the code I have for validation need something else in order to hook into the AngularJS? I followed what was already there. Thanks for looking!

Reply:
@emmagarland, Your code seems to be correct. I hope to find the issue today, with some help from a frontend developer 🤞

Reply:
Thanks @bergmania for letting me know, fingers crossed!

Reply:
Resolving as related to a general frontend issue to be resolved separately (will add ID once pushed)